<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileWindowNavigator</class>
 <widget class="QWidget" name="FileWindowNavigator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>32</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="backBtn">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../resource/resource.qrc">
       <normaloff>:/icons/arrow_left.png</normaloff>:/icons/arrow_left.png</iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="forwardBtn">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../resource/resource.qrc">
       <normaloff>:/icons/arrow_right.png</normaloff>:/icons/arrow_right.png</iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="refreshBtn">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../resource/resource.qrc">
       <normaloff>:/icons/reflash.png</normaloff>:/icons/reflash.png</iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="homeBtn">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../resource/resource.qrc">
       <normaloff>:/icons/home.png</normaloff>:/icons/home.png</iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>路径:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="pathLied">
     <property name="styleSheet">
      <string notr="true">border-width:1px;
border-radius:4px;
border:1px solid gray;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font-size:14px;
</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="goBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Go</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
